health/micro-ui/web/micro-ui-internals/example/src/UICustomizations.js

```
@@ -765,176 +765,177 @@
+              );
+            } else {
+              return (
+                <div>
+                  <p>{t("NA")}</p>
+                </div>
+              );
+            }
+  
+          case "MICROPLAN_STATUS":
+            if (value && value != "NA") {
+              return <p>{t(Digit.Utils.locale.getTransformedLocale("MICROPLAN_STATUS_" + value))}</p>;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)